### PR TITLE
fix: Adding missing alt tags.

### PR DESCRIPTION
**Pull request by Chakravarthy7102**
## Summary of change
Added missing alt tags to images in some pages and removed duplicate alt tag description with appropriate ones.

## Related issues
- Link to issue1 here
- Link to issue1 here

## Checklist
- [ ] Algolia search needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Sitemap needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Checked for broken links? (Run `cd v2 && MODE=production npx docusaurus build`)
- [ ] Changes required to the demo apps corresponding to the docs?

## Remaining TODOs for this PR
- [ ] Item1
- [ ] Item2

### CODE REVIEW

File: v2/emailpassword/serverless/with-aws-lambda/setup-lambda-layer.mdx [hunk 1]
```diff
@@ -42,18 +42,18 @@ zip -r supertokens-python.zip python/
 
 ## 2) Upload SuperTokens lambda layer
 - Open AWS Lambda dashboard and click on layers:
-<img src="/img/integration-lambda/sidebar.png" height="300" />
+<img src="/img/integration-lambda/sidebar.png" alt="AWS Lamda sidebar UI" height="300" />
```
Comment (prateek3255) [inline v2/emailpassword/serverless/with-aws-lambda/setup-lambda-layer.mdx:45]:
```suggestion
<img src="/img/integration-lambda/sidebar.png" alt="AWS Lambda sidebar UI" height="300" />
```

File: v2/emailpassword/serverless/with-aws-lambda/setup-lambda-layer.mdx [hunk 2]
```diff
@@ -42,18 +42,18 @@ zip -r supertokens-python.zip python/
...
 
 - Click "Create Layer" button:
-<img src="/img/integration-lambda/create-layer.png" width="600" />
+<img src="/img/integration-lambda/create-layer.png" alt="Create layer button UI" width="600" />
 
 - Give a name for your layer, upload the zip and select the runtime
 <BackendSDKTabs disableGolang>
   <TabItem value="nodejs">
-    <img src="/img/integration-lambda/node-lambda-layer.png"/>
+    <img src="/img/integration-lambda/node-lambda-layer.png" alt="Lamda layer node configuration UI"/>
```
Comment (prateek3255) [inline v2/emailpassword/serverless/with-aws-lambda/setup-lambda-layer.mdx:53]:
```suggestion
    <img src="/img/integration-lambda/node-lambda-layer.png" alt="Lambda layer node configuration UI"/>
```

File: v2/emailpassword/serverless/with-aws-lambda/setup-lambda.mdx [hunk 3]
```diff
@@ -20,27 +20,27 @@ import TabItem from '@theme/TabItem';
 - Click "Create Function" in the AWS Lambda dashboard, enter the function name and runtime, and create your Lambda function.
 <BackendSDKTabs disableGolang>
   <TabItem value="nodejs">
-    <img src="/img/integration-lambda/create-function-node.png"/>
+    <img src="/img/integration-lambda/create-function-node.png" alt="Create new Lambda configurations UI Node"/>
   </TabItem>
   <TabItem value="python">
-    <img src="/img/integration-lambda/create-function-python.png"/>
+    <img src="/img/integration-lambda/create-function-python.png" alt="Create new Lamda configurations UI Python"/>
```
Comment (prateek3255) [inline v2/emailpassword/serverless/with-aws-lambda/setup-lambda.mdx:26]:
```suggestion
    <img src="/img/integration-lambda/create-function-python.png" alt="Create new Lambda configurations UI Python"/>
```

Comment (prateek3255) [reply v2/emailpassword/serverless/with-aws-lambda/setup-lambda.mdx:26]: Okay I see similar typos in all of the changed file as well can you fix that?

File: v2/community/reusableMD/supported-tech-stacks-backend.mdx [hunk 4]
```diff
@@ -5,7 +5,7 @@
   justifyContent: "start",
   flexWrap: "wrap"
 }}>
-<img src="/img/logos/node.svg" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/python.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/go.svg" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
+<img src="/img/logos/node.svg" alt="nodejs logo" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```
Comment (prateek3255) [inline v2/community/reusableMD/supported-tech-stacks-backend.mdx:8]:
```suggestion
<img src="/img/logos/node.svg" alt="Node.js logo" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```

Should ideally use the case used on their official website for all the logos

File: v2/community/reusableMD/supported-tech-stacks-frontend-custom-ui.mdx [hunk 5]
```diff
@@ -5,12 +5,12 @@
...
 }}>
-<img src="/img/logos/react.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/vue.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/angular.svg" style={{width: "65px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/android.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/ios.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/electron.svg" style={{width: "60px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/capacitor-js.svg" style={{width: "60px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/js_logo.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
+<img src="/img/logos/react.svg" alt="react logo" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```
Comment (prateek3255) [inline v2/community/reusableMD/supported-tech-stacks-frontend-custom-ui.mdx:8]:
```suggestion
<img src="/img/logos/react.svg" alt="React logo" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```
Make similar changes for other logos in the file as well

File: v2/community/reusableMD/supported-tech-stacks-frontend.mdx [hunk 6]
```diff
@@ -5,12 +5,12 @@
   justifyContent: "start",
   flexWrap: "wrap"
 }}>
-<img src="/img/logos/react.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/vue.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/angular.svg" style={{width: "65px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/electron.svg" style={{width: "60px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/capacitor-js.svg" style={{width: "60px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/js_logo.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
+<img src="/img/logos/react.svg" alt="react logo" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```
Comment (prateek3255) [inline v2/community/reusableMD/supported-tech-stacks-frontend.mdx:8]: And here as well

**Review by prateek3255 — CHANGES_REQUESTED** [review]

File: v2/community/reusableMD/supported-tech-stacks-frontend.mdx (hunk 6 above)
Comment (Chakravarthy7102) [reply v2/community/reusableMD/supported-tech-stacks-frontend.mdx:8]: done

File: v2/community/reusableMD/supported-tech-stacks-frontend-custom-ui.mdx (hunk 5 above)
Comment (Chakravarthy7102) [reply v2/community/reusableMD/supported-tech-stacks-frontend-custom-ui.mdx:8]: done

File: v2/community/reusableMD/supported-tech-stacks-backend.mdx (hunk 4 above)
Comment (Chakravarthy7102) [reply v2/community/reusableMD/supported-tech-stacks-backend.mdx:8]: done

File: v2/emailpassword/serverless/with-aws-lambda/setup-lambda.mdx (hunk 3 above)
Comment (Chakravarthy7102) [reply v2/emailpassword/serverless/with-aws-lambda/setup-lambda.mdx:26]: done

File: v2/emailpassword/serverless/with-aws-lambda/setup-lambda-layer.mdx (hunk 2 above)
Comment (Chakravarthy7102) [reply v2/emailpassword/serverless/with-aws-lambda/setup-lambda-layer.mdx:53]: done

**Review by prateek3255 — APPROVED** [review]

**Pull request merged**